libs/sdk-ui-kit/src/Dialog/ConversionTouchPointDialogBase.tsx

```diff
@@ -0,0 +1,130 @@
+// (C) 2021 GoodData Corporation
+import React, { useEffect, useState } from "react";
+import { useIntl } from "react-intl";
+import { useHubspotForm } from "@aaronhayes/react-use-hubspot-form";
+import { LoadingComponent } from "@gooddata/sdk-ui";
+
+import { Button } from "../Button";
+import { HubSpotFormLocale } from "@aaronhayes/react-use-hubspot-form/build/main/hubspot-form-locale.enum";
+
+declare global {
+    interface Window {
+        jQuery: object;
+        hbspt: undefined | { isSuccessMessageShow: boolean };
+    }
+}
+
+/**
+ * @internal
+ */
+export interface HubspotFormField {
```

Comment:
Should we add a prefix `I` to the interface name, to make it consistent with other interfaces?
`HubspotFormField` → `IHubspotFormField`

Reply:
Fixed. Thanks!